google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/FlexibleFacetEditor.java

```diff
@@ -271,13 +273,27 @@ public void onFacetInitialized(@NotNull Facet facet) {
       ((AppEngineFlexibleFacet) facet).getConfiguration().setDockerDirectory(
           dockerDirectoryField.getText());
     }
+
+    // Called on explicitly adding the facet through Project Settings -> Facets, but not on the
+    // Framework discovered "Configure" popup.
+    UsageTrackerProvider.getInstance()
```

Comment:
since you've been updating other equivalent calls for standard, should you do it for the ones in this class too?